tests/test_edgeql_ddl.py

```diff
@@ -879,6 +879,13 @@ async def test_edgeql_ddl_26(self):
                 ALTER TYPE Child ALTER LINK target DROP OWNED;
             """)
 
+    async def test_edgeql_ddl_27(self):
+        # Test that identifiers that are SQL keywords get quoted.
+        # Issue 1667
+        await self.con.execute("""
+            CREATE TYPE test::Foo {CREATE LINK left -> test::Foo};
```

Comment:
Can you add few more links, like a edgeql-quoted `CREATE` and `LINK`, for example.

Reply:
OK, added a few more names.